### ISSUE TITLE
XML event binding doesn't work in a repeater which is inside a TabView

### ISSUE THREAD

**Issue by lyonzy** [NativeScript/NativeScript#1344]
When I do something like this:
```xml
<TabView>
	<TabView.items>
		<TabViewItem title="Buttons">
			<TabViewItem.view>
				<ScrollView>
					<Repeater items="{{ items }}">
						<Repeater.itemsLayout>
							<StackLayout orientation="vertical" />
						</Repeater.itemsLayout>
						<Repeater.itemTemplate>
							<StackLayout loaded="buttonLoaded">
								<Button text="test"  />
							</StackLayout>
						</Repeater.itemTemplate>
					</Repeater>
				</ScrollView>
			</TabViewItem.view> 
		</TabViewItem>
	</TabView.items>
</TabView>
```
And then load `items` asynchronously, the `buttonLoaded` function doesn't get bound to the Page's code. This is because when `getExports` inside `builder` traverses up the instance's parents, somewhere around the TabView a view has an undefined parent, so `getExports` returns `undefined`. 

Is an `undefined` parent expected behaviour/a known issue for TabViews?

It seems this doesn't happen during the initial page load because a reference to `exports` is kept separately of the View tree.

**Comment by lyonzy** [comment]
Upon further investigation, it seems that the view inside the tab (in this case the `ScrollView`) doesn't have a parent.

**Comment by nsndeck** [comment]
The issue has been fixed with this commit #1361. Now `getExports` will return valid results.

**Comment by lyonzy** [comment]
Awesome, thanks :)
